examples/deployment/README.md

```diff
@@ -0,0 +1,33 @@
+# ASA deployment using [Algorand Builder](https://github.com/scale-it/algorand-builder/)
+
+## Usage
+### Create your local network:
+https://developer.algorand.org/tutorials/create-private-network/
+
+### Start whole network:
+`goal network start -r ~/.algorand-local/`
+
+### Start/stop a single node:
+`goal node start -d ~/.algorand-local/Node/`
+
+### Currently it needs to be linked with algob:
+`git clone https://github.com/scale-it/algorand-builder/`
+`cd algorand-builder/packages/algob`
+`yarn`
+`yarn link`
+
+### Change your keys
+Use your editor to edit `algob.config.js`
+
+### Run
+`algob deploy`
+`algob run scripts/query/john-balances.js`
+
+## Authors
+- robert-zaremba
+- Invertisment
+- amityadav0
+- as04
```

Comment:
Do we need this at all? If you want it then you should put yourself first, or use alphabetic order and remove `as04` - there is 0 contribution / working code from her.

Reply:
I think I should put something in my own repository. 
Will remove from both places then.